src/connectService.ts

```diff
@@ -46,6 +55,13 @@ const rpcError = (id: string | number) => {
   return Object.assign(rpcError.toObject(), { id })
 }
 
+const jwt_decode = <T>(jwt: string): T => {
```

Comment:
Could be named `decodeJWT` maybe to be more consistent with other function names?